src/components/Header/SideNav/index.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react'
+import styled from 'styled-components'
+import remcalc from 'remcalc'
+import Flex from 'styled-flex-component'
+import is from 'styled-is'
+
+import CloseButton from './CloseButton'
+import Dropdown from './Dropdown'
+import OuterAnchorItem from './OuterAnchorItem'
+
+const SideNavList = styled.ul``
+
+const SideNavPanel = styled.div.attrs({
```

Comment:
why not `styled.nav`?

Reply:
Yeah, nice :+1: I just copy/pasted [the code from the old file](https://github.com/yldio/yld.io/pull/157/files#diff-3af6432a615e0c766f3ca19b44ffa57fL85). Lazy lazy :sleeping: